js/components/portal-dashboard/all-responses-popup/popup-header.tsx

```diff
@@ -32,7 +34,9 @@ export class PopupHeader extends React.PureComponent<IProps, IState>{
   }
 
   private renderHeaderLeft = () => {
-    const activityName = "Hurricane Module V2 Activity 1: Hurricane Risk";
+    const { currentActivity } = this.props;
+    const activityName = currentActivity?.get("name");
+    // const activityName = "Hurricane Module V2 Activity 1: Hurricane Risk";
     return (
       <div className={css.headerLeft}>
         <AssignmentIcon className={`${css.assignmentIcon} ${css.icon}`} />
```

Comment:
Down below in renderHeaderRight, I would clean up code and use:
`const closeIconClass = `${css.closeIcon} ${this.state.inFeedbackMode ? css.inFeedbackMode : ""}`;`

Reply:
And I would change in renderResponseFeedbackToggle to:
```
<div className={css.toggleResponsesFeedback}>
```

Reply:
And in renderResponseFeedbackToggle :
```
const responsesToggleClass = `${css.responseToggle} ${this.state.inFeedbackMode ? css.toggleOff : ""}`;
```